gaps/gap-6_tech_council_process/gap-6_tech_council_process.md

```diff
@@ -0,0 +1,79 @@
+---
+gap: 6
+title: Tech Council process
+description: Rules of operation for Golem Tech Council
+author: stranger80 (@stranger80)
+status: Draft
+type: Meta
+---
+
+## Abstract
+This is proposal to structure the proceedings of Golem Tech Council in order to streamline the flow of agenda items and make the approval process efficient.
+
+## Motivation
+The purpose is to ensure the Tech Council operates efficiently, with as few synchronous actions (like meetings) as possible, and with transparency to all Golem Factory team members.
+
+## Specification
+
+The main purpose of this proposed process is to make proceedings of Tech Council items more efficient and asynchronous (eg. have them raised and reviewed offline, and only use Tech Council sessions to discuss questions on items which have been already digested by participants). 
+
+This is why responsibilities of the Facilitator also must include upfront facilitation of Tech Council session agenda, so that the participants aregiven an early heads-up and are able to prepare for the session (eg. by reading and reviewing items proposed on the agenda).
+
+This section specifies the rules for Tech Council proceedings, by defining key elements and workflow.
+
+### Tech Council Members
+
+Two roles of Tech Council members are specified:
+- **Participant** - participant of Tech Council proceedings and sessions
+- **Facilitator** - responsible for:
+  - reviewing and managing the Kanban board and GAP board
+  - preparing agenda for Tech Council sessions
+  - facilitating Tech Council sessions
+
+### Tech Council Kanban
+
+A Notion board where all Tech Council items are recorded and their status indicated.
+
+### Raising items for Tech Council
+
+The topics for discussion by Tech Council can be raised by any Tech Council member, in a form of:
+- **A card on Tech Council Kanban** 
+  
+  **NOTE**: the card raised on Kanban should include following elements at minimum:
+  - Indicative title
+  - Description sufficient to set the context and topic of the item in detail allowing for readers to understand and build opinion on the matter.
+  - Indication of expected input from Tech Council (ie. **what is Tech Council expected to do with this item?**)
+
+- **A GAP item**
+
+  Just raise a GAP as per [GAP Process](../gap-1_gap_process/gap-1_gap_process.md)
+
+The newly raised topics shall be reviewed by the **Facilitator** and included in the agenda for Tech Council session (wth comments if relevant).
+
+### Tech Council Session
+
+A fortnightly (once per 2 weeks) Tech Council session is an opportunity for participants to address the questions which have been raised against the Tech Council items. 
+
+- A Tech Council session **agenda** is prepared by the **Facilitator** and announced to all participants **at least 2 days prior** to the scheduled session.
```

Comment:
via email & instant messenger #tech-council channel for all interested ppl within GF to be able to join (meeting details) / comment (agenda point URLs)

Reply:
OK,added.